### PR TITLE
fix example for allow_nil

### PR DESCRIPTION
**Pull request by ppraisethesun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -354,8 +354,8 @@ defmodule YourOperation do
   def process(params), do: params
 end
 
-{:ok, %{a: 1}} = YourOperation.run(a: 1)
-{:ok, %{a: nil}} = YourOperation.run(a: nil)
+{:ok, %{a: 1}} = YourOperation.run(a: 1, b: 1)
+{:ok, %{a: nil}} = YourOperation.run(a: nil, b: 1)
 {:ok, %{b: 1}} = YourOperation.run(b: 1)
```
Comment (madeinussr) [inline README.md]: those two:
```
{:ok, %{b: 1}} = YourOperation.run(b: 1)
{:error, {:validation, %{b: ["has wrong type"]}}} = YourOperation.run(b: nil)
```
are broken as well

Comment (ppraisethesun) [reply README.md]: yeah, missed it

File: README.md [hunk 2]
```diff
@@ -354,10 +354,10 @@ defmodule YourOperation do
...
 end
 
-{:ok, %{a: 1}} = YourOperation.run(a: 1)
-{:ok, %{a: nil}} = YourOperation.run(a: nil)
-{:ok, %{b: 1}} = YourOperation.run(b: 1)
-{:error, {:validation, %{b: ["has wrong type"]}}} = YourOperation.run(b: nil)
+{:ok, %{a: 1}} = YourOperation.run(a: 1, b: 1)
+{:ok, %{a: nil}} = YourOperation.run(a: nil, b: 1)
+{:ok, %{b: 1}} = YourOperation.run(a: nil, b: 1)
+{:error, {:validation, %{b: ["has wrong type"]}}} = YourOperation.run(a: nil, b: nil)
```
Comment (madeinussr) [inline README.md]: `{:error, {:validation, %{b: ["doesn't allow nil", "has wrong type"]}}} = YourOperation.run(a: nil, b: nil)` 🙏

**Comment by madeinussr** [comment]
cool, thank you! 👍

**Pull request merged**